### ISSUE TITLE
Any further plans?

### ISSUE THREAD

**Issue by strich** [Thocook/Arbitrage_Finder#1]
This looks really ace! Nice work. Just curious if you plan to take this any further at this point?

**Comment by Thocook** [comment]
Thanks glad you like it! I might expand on this in the future, depends if I have the need for it. That plan would look something along the lines of getting a large list of products within a given category (bathroom tech for a simple example) then checking all of them against Alibaba comparables to create a data set of all of the best arbitrage opportunities for that niche. Along with this I would probably add more sites like dhgate and Wish. Also since the data set of supplies would be bigger I would refine the criteria so that higher end suppliers with faster shipping would be favoured.

**Comment by strich** [comment]
Yeah nice ideas! I've never developed with Juptyr before but I might contribute if you like.
Do you know if it's possible to gain any insight on how much unit movement an Amazon category is having and therefore that item type?
